chancloser.go

```diff
@@ -347,6 +347,19 @@ func (c *channelCloser) ProcessCloseMsg(msg lnwire.Message) ([]lnwire.Message, b
 				"instead have %v", spew.Sdump(msg))
 		}
 
+		// TODO(roasbeef): if channel responder and frozen, do nothing
```

Comment:
This seems to already be the case, can it be removed?

Reply:
Yep, just a TODO left over from when I was planning out this PR.